test/docker.bats

```diff
@@ -73,6 +73,7 @@ load test_helpers
 		-v /var/run/docker.sock:/tmp/docker.sock:ro \
 		-v $BATS_TEST_DIRNAME/../nginx.tmpl:/etc/docker-gen/templates/nginx.tmpl:ro \
 		--volumes-from bats-nginx \
+		--expose 80 \
```

Comment:
@md5 See [this comment](https://github.com/jwilder/nginx-proxy/pull/337#issuecomment-200768674).